### DISCUSSION TITLE
do not add `is:open` as default on `Question / Answer` Categories

### DISCUSSION THREAD

**Discussion by scarf005** [community/community#56490] [Discussions]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/54838975/b2ccbae2-2418-4973-b7a1-f6ff08137b26)
for github discussions, `is:open` tag is default appended to search.

![image](https://github.com/community/community/assets/54838975/6ffc74ee-717a-4743-a3f3-aa9e46ef8235)

however, having `is:open` is cumbersome for `Question / Answer` because answered questions are by default **closed**.
![image](https://github.com/community/community/assets/54838975/dd6821a4-941d-4440-9e6a-80fdbac9e6e2) ![image](https://github.com/community/community/assets/54838975/6762eea6-4786-41d1-9d3b-d77a4135be53)

**Comment by Jake7208** [comment]
Seems Legit.
